src/pybind/mgr/dashboard/frontend/src/app/ceph/cephfs/cephfs-clients/cephfs-clients.component.ts

```diff
@@ -14,10 +23,30 @@ export class CephfsClientsComponent implements OnInit {
   @Input()
   id: number;
 
+  permission: Permission;
+  tableActions: CdTableAction[];
+  modalRef: BsModalRef;
   clients: any;
   viewCacheStatus: ViewCacheStatus;
+  selection = new CdTableSelection();
 
-  constructor(private cephfsService: CephfsService, private i18n: I18n) {}
+  constructor(
+    private cephfsService: CephfsService,
+    private modalService: BsModalService,
+    private notificationService: NotificationService,
+    private authStorageService: AuthStorageService,
+    private i18n: I18n,
+    private actionLabels: ActionLabelsI18n
+  ) {
+    this.permission = this.authStorageService.getPermissions().cephfs;
+    const evictAction: CdTableAction = {
+      permission: 'update',
+      icon: 'fa-sign-out',
```

Comment:
Please use the new Icons class
```suggestion
      icon: Icons.xxx,
```